vyper/semantics/types/function.py

```diff
@@ -388,6 +456,34 @@ def compare_signature(self, other: "ContractFunctionT") -> bool:
 
         return True
 
+    def get_default_value(self, kwarg_name: str) -> Optional[vy_ast.VyperNode]:
```

Comment:
this is fine but for perf reasons i would prefer to see a `cached_property` like `default_values` which returns a dict.

Reply:
done in7f8985cb21a3dc8d4fa83637cea96c6534371c2b